flambe/experiment/options.py

```diff
@@ -131,3 +131,22 @@ def __repr__(self) -> str:
     @classmethod
     def to_yaml(cls, representer: Any, node: Any, tag: str) -> Any:
         return representer.represent_sequence('!g', node.elements)
+
+
+@alias('cluster')
```

Comment:
I would recommend the name remote over cluster since we may some day send to a machine and not a cluster

Reply:
Ok! I don't have strong opinion about this naming.

Reply:
Done

Reply:
I'll bing this up again. When using a single remote machine, that will be also a "cluster" (of a single machine). That implementation will most likely follow our cluster paradigm (the orchestrator and factory will be the same machine, but logically if will be separated).

I think `cluster` is a better option here to keep consistency.

Reply:
@nmatthews-asapp opinion?

Reply:
Discussed offline. `!cluster` for now!